model/Service/Properties/serviceHostingCountry.md

```diff
@@ -18,4 +18,4 @@ See the [ISO 3166-1 alpha-3 Wikipedia page](https://en.wikipedia.org/wiki/ISO_31
 
 - name: serviceHostingCountry
 - Nature: DataProperty
-- Range: CountryCodeAlpha3
+- Range: Core/CountryCodeAlpha3
```

Comment:
```suggestion
- Range: /Core/CountryCodeAlpha3
```

Need the leading slash.

See for example: https://github.com/spdx/spdx-3-model/blob/8caa3fb87d53b356b211a64aca5f6901e14be441/model/SimpleLicensing/Properties/licenseListVersion.md?plain=1#L22-L26